ivy/functional/frontends/torch/Utilities.py

```diff
@@ -0,0 +1,10 @@
+#local
+from ast import Return
+import ivy
+
+def result_type(tensor1, tensor2):
+    Return 
```

Comment:
This is not how things are returned in python. This should be lower case, and the thing being returned should be on the same line